core/store/migrate/migrations/0150_gaslaneprice_vrf.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+ALTER TABLE vrf_specs
+    ADD COLUMN "gas_lane_price" NUMERIC(78, 0)
```

Comment:
is 78 digits special for any reasons?

Reply:
Yes, its large enough to hold the max uint256